packages/utilities/src/memoize.test.ts

```diff
@@ -1,10 +1,10 @@
 import { memoize, memoizeFunction, setMemoizeWeakMap } from './memoize';
-import * as weakMapPolyfill from 'es6-weak-map/polyfill';
+import weakMapPolyfill = require('es6-weak-map');
```

Comment:
@dzearing, is this ok? I was having a hard time getting the /polyfill path to work because it doesn't have a d.ts file. I remember you mentioning that this was a fix for UT environments? Based on [this](https://github.com/medikoo/es6-weak-map), it seems like the behavior is different for the /polyfill path.